test/python/visualization/test_circuit_latex.py

```diff
@@ -699,6 +699,26 @@ def test_wire_order(self):
         )
         self.assertEqualToReference(filename)
 
+    def test_wire_order_only_qubits(self):
+        """Test the wire_order list with qubits only to latex drawer"""
+        filename = self._get_resource_path("test_wire_order_only_qubits.tex")
+        qr = QuantumRegister(4, "q")
+        cr = ClassicalRegister(4, "c")
+        cr2 = ClassicalRegister(2, "ca")
+        circuit = QuantumCircuit(qr, cr, cr2)
+        circuit.h(0)
+        circuit.h(3)
+        circuit.x(1)
+        circuit.x(3).c_if(cr, 12)
+        circuit_drawer(
+            circuit,
+            cregbundle=False,
+            wire_order=[2, 1, 3, 0, 4, 5, 6, 7, 8, 9],
+            filename=filename,
+            output="latex_source",
+        )
+        self.assertEqualToReference(filename)
```

Comment:
This test says its `wire_order` is only qubits, but it's actually got all of them.  It should probably be a test of what it says, and shouldn't include the clbits.

Reply:
actually, these tests make little sense now. Removing them in 445489c4a8832d9fd73f5a155d21ec3c96f07084 and add those that cover the involved code in this PR.